x-pack/plugins/session_view/public/components/process_tree/index.test.tsx

```diff
@@ -38,6 +38,54 @@ describe('ProcessTree component', () => {
       expect(renderResult.queryAllByTestId('sessionView:processTreeNode')).toBeTruthy();
     });
 
+    it('When Verbose mode is OFF, it should not show all childrens', () => {
+      renderResult = mockedContext.render(
+        <ProcessTree
+          sessionEntityId="3d0192c6-7c54-5ee6-a110-3539a7cf42bc"
```

Comment:
instead of using a hardcoded string, we should get the `sessionEntityId` from `mockData` e.g. something like`mockData[0].events[0]` in case the mock data changes in the future. I updated the other ones in another PR, just need to do it for the new tests you added.